### ISSUE TITLE
The `upload-time` and `uploader` package resources refer to the latest revision and admin who did it.

### ISSUE THREAD

**Issue by Kleidukos** [haskell/hackage-server#1145]
The package sub-resources for upload-time and uploader incorrectly refer to the latest revision and Hackage admin who performed the revision. 

For example:

https://hackage.haskell.org/package/AC-Vector-2.3.2

* Uploaded: by AndrewCoppin at 2011-08-12T12:33:08Z
* HerbertValerioRiedel at 2018-09-30T13:49:35Z

* https://hackage.haskell.org/package/AC-Vector-2.3.2/uploader mentions HerbertValerioRiedel
* https://hackage.haskell.org/package/AC-Vector-2.3.2/upload-time mentions 2018-09-30T13:49:35Z

**Comment by andreasabel** [comment]
@Kleidukos Did you find / can you blame the responsible commit?  Maybe there is a comment there whether this is intended behavior or not...

**Comment by Kleidukos** [comment]
@andreasabel There is no "responsible" commit to blame, I believe it was always the behaviour since at least 2013, but it's not a correct behaviour.

**Comment by gbaz** [comment]
I agree that this is an overt and easy fix -- will bring resources in line with the displayed info on the standard page.
